packages/libs/react-ui/src/components/Select/Select.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { SystemIcon } from '../Icons';
+
+import { Option } from './Option';
+import { ISelectProps, Select } from './Select';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import React, { useState } from 'react';
+
+const meta: Meta<
+  {
+    icon: keyof typeof SystemIcon;
+  } & ISelectProps
+> = {
+  title: 'Components/Select',
+  argTypes: {
+    disabled: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    icon: {
+      options: [
+        undefined,
+        ...(Object.keys(SystemIcon) as (keyof typeof SystemIcon)[]),
+      ],
+      control: {
+        type: 'select',
+      },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<
+  {
+    icon: keyof typeof SystemIcon;
+  } & Omit<ISelectProps, 'icon'>
+>;
+
+export const Dynamic: Story = {
+  name: 'Select',
+  args: {
+    icon: undefined,
```

Comment:
can you add a value option.
So that people can see that you can set the default value